utils/AuthService.js

```diff
@@ -0,0 +1,62 @@
+export default class AuthService { 
+    constructor(domain) { 
+        this.domain = domain || 'http://localhost:5000' 
+        this.fetch = this.fetch.bind(this) 
```

Comment:
Que ondas puedes usar la arrow function  para no tener que hacer el bind(this). 
  fetch = (url, options) => { } , con esto ya te trae el la funcion con el binding respectivo :)

Reply:
Si mejor usar arrows